### DISCUSSION TITLE
Spread CONTRIBUTE.md where ever it makes sense

### DISCUSSION THREAD

**Discussion by cuihtlauac** [ocaml/ocaml.org#1258] [Ideas]
Let's explain this idea with an example. Consider `/jobs` 

The web page reflects file:

https://github.com/ocaml/ocaml.org/blob/main/data/jobs.yml

Per:

https://github.com/ocaml/ocaml.org/blob/main/CONTRIBUTING.md#content-job

We expect people to file PR in `jobs.yml` when they want something to appear on https://ocaml.org/jobs

The idea is either:
- Have the text paragraph on jobs from `CONTRIBUTING.md` inside the `/jobs` page
- Have a link from `https://ocaml.org/jobs` to `https://github.com/ocaml/ocaml.org/blob/main/CONTRIBUTING.md#content-job`
- Have a link from `https://ocaml.org/jobs` to `https://github.com/ocaml/ocaml.org/blob/main/data/jobs.yml`

The rest of the idea is to apply the same logic to all the pages where we rely on third-party contributions

Applicable areas:
- [ ] Blog & news
- [ ] Jobs
- [ ] Success Stories
- [ ] Academic Users
- [ ] Industrial Users
- [ ] Ocaml Books
- [ ] Communtity Events
- [ ] Featured Packages
- [ ] Changelog

**Comment by sabine** [comment]
For linking to the corresponding paragraph in `CONTRIBUTING.md`, and, from there, linking to the corresponding file from there, the impact vs effort ratio and incurred code complexity looks very good to me.

**Comment by christinerose** [comment]
[I opened a PR](https://github.com/ocaml/ocaml.org/pull/1323) with some suggested edits and questions about clarity for CONTRIBUTING.md.
